src/command.h

```diff
@@ -183,6 +187,52 @@ class DrawRectCommand : public PipelineCommand {
   float height_ = 0.0;
 };
 
+/// Command to draw a grid of recrangles on screen.
+class DrawGridCommand : public PipelineCommand {
+ public:
+  explicit DrawGridCommand(Pipeline* pipeline, PipelineData data);
```

Comment:
explicit is only needed if there is a single parameter

Reply:
copied off draw_rect. Since the number of parameters here dropped to one, I left the explicit. Not fixed in draw_rect (or other calls here, which also have the explicit keyword).